client/wfnews-war/src/main/angular/src/app/components/admin-incident-form/admin-incident-form.component.ts

```diff
@@ -141,14 +139,76 @@ export class AdminIncidentForm implements OnInit, OnChanges {
     // TODO: This can be removed once the onInit is updated to map the form correctly
     if (changes.adminIncident && changes.adminIncident.currentValue){
       this.currentAdminIncident = changes.adminIncident.currentValue
-      // We need to load the Published Incident from IM as well at this point!
-      this.incidentForm.patchValue(this.currentAdminIncident)
-
-      // update the stub, until we wire this properly
-      this.incident.wildfireYear = this.wildFireYear
-      this.incident.incidentNumberSequence = this.currentAdminIncident.incidentNumberSequence
-      this.incident.fireName = this.currentAdminIncident.incidentName
-      this.incident.incidentData = this.currentAdminIncident
+      var self = this;
+      
+      this.publishedIncidentService.fetchIMIncident(this.currentAdminIncident.wildfireYear.toString(), this.currentAdminIncident.incidentNumberSequence.toString())
+          .subscribe(incidentResponse => {
+                let publishedIncident = incidentResponse.response;
+                this.publishedIncidentType = publishedIncident.type;
+                self.incident.fireNumber = self.currentAdminIncident.incidentNumberSequence;
+                self.incident.wildfireYear = self.currentAdminIncident.wildfireYear;
+                self.incident.incidentNumberSequence= self.currentAdminIncident.incidentNumberSequence;
+                self.incident.fireName = publishedIncident.incidentName;
+                self.incident.publishedStatus = "DRAFT",
+                self.incident.location = publishedIncident.incidentLocation.geographicDescription;
+                
+                this.publishedIncidentDetailGuid = this.currentAdminIncident["wildfireIncidentGuid"];
+                this.incidentForm.patchValue(this.incident);
+                this.incidentForm.patchValue(this.currentAdminIncident);
+
+                this.cdr.detectChanges();
+
+                incidentResponse.getPublishedIncident.subscribe( 
+                  (response)=> {
+                    self.publishedIncidentDetailGuid = response.publishedIncidentDetailGuid;
+                    self.incident.traditionalTerritory = response.traditionalTerritoryDetail;
+                    self.incident.lastPublished = publishedIncident.publishedTimestamp,
+                    self.incident.location = response.incidentLocation;
+                    self.incident.sizeType = response.incidentSizeType == "null" ? 1 : Number(response.incidentSizeType);
+                    if(self.incident.sizeType == 1)
```

Comment:
Not important for now, but size is being removed from publishedIncident and we'll just use the incidents value